### ISSUE TITLE
.NET sample

### ISSUE THREAD

**Issue by lionello** [defang-io/defang#294]

**Comment by Bidombamb** [comment]
yeah uh, i think your supposed to add details about what's the problem you are having now?

**Comment by lionello** [comment]
> yeah uh, i think your supposed to add details about what's the problem you are having now?

Yes, very true! My bad. We're lacking a C# Sample, that's what this issue was for.
